tests/topotests/ldp-oc-topo1/test_ldp_oc_topo1.dot

```diff
@@ -0,0 +1,76 @@
+## Color coding:
```

Comment:
aren't the pdf files generated as part of .dot compilation?  Why are we including them in the repo?

Reply:
I will remove the pdf files from these 2 new topo tests.

A few other topotests have the pdf files checked in.   I assumed it was recommended to checkin the pdf files.